src/io/jesi/customs/strict/kaocha.clj

```diff
@@ -0,0 +1,27 @@
+(ns io.jesi.customs.strict.kaocha
+  (:refer-clojure :exclude [remove-ns])
+  (:require
+    [com.rpl.specter :as sp]
+    [io.jesi.backpack :as bp]
+    [io.jesi.backpack.macros :refer [def-]]
+    [kaocha.plugin :refer [defplugin]]))
+
+(def- strict-ns (str 'io.jesi.customs.strict))
```

Comment:
```suggestion
(def- strict-ns "io.jesi.customs.strict")
```
Less characters. Does the quoted symbol give us any other benefit?

Reply:
Having it quoted means that Cursive can link to the ns (cmd+b). Helps prevent typos